.github/workflows/bundler-audit.yml

```diff
@@ -36,4 +36,4 @@ jobs:
           bundler-cache: true
 
       - name: Run bundler-audit
-        run: bundle exec bundler-audit check --update
+        run: bin/bundler-audit check --update
```

Comment:
When I'm operating in a local CLI environment, it's always satisfying to just type "bin something" and have it run a bundle-version-appropriate command for me. It's nice to see that technology coming to this PR.